src/Core/Enums/EventType.cs

```diff
@@ -48,5 +48,7 @@ public enum EventType : int
         // Organization_ClientExportedVault = 1602,
 
         Policy_Updated = 1700,
+
+        SsoUser_Deleted = 1800
```

Comment:
You'll need to add support for this in the web vault now.

Reply:
Let's re-clasify this event id under `OrganizationUser_UnlinkedSso = 1505,`. See further comment below.

Reply:
Done